File: src/frontend/my-RacksStand/src/app/suppliers/shared/supplier-model.ts
export class SupplierModel {
    Id: string;
    CompanyId: string;
    Title: string;
    FirstName: string;
    MiddleName: string;
    LastName: string;
    Email: string;
    Company: string;
    Phone: string;
    Mobile: string;
    Fax: string;
    PanNo: string;
    Website: string;
    Bio: string;
    Status: number;
    CreatedOn: Date;
    CreatedBy: string;
    UpdatedOn: Date;
    UpdatedBy: string;
    Addresses: SupplierAddressModel[];
}
export class SupplierAddressModel {
    Id: string;
    City: string;
    Country: number;
    SupplierId: string;
    Email: string;
    Fax: string;
    Phone: string;
    Remark: string;
    State: string;
    Status: number;
    Street: string;
    ZipCode: string;

}

File: src/backend/BusinessLogic/Models/Core/SupplierModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models.Core
{
    [Serializable]
    public class SupplierModel
    {
        public SupplierModel()
        {
            this.Addresses = new List<SupplierAddressModel>();
        }
        [Required]
        [MaxLength(32)]
        public string Id { get; set; }
        [Required]
        [MaxLength(50)]
        public string FirstName { get; set; }
        [Required]
        [MaxLength(50)]
        public string LastName { get; set; }
        [MaxLength(50)]
        public string MiddleName { get; set; }
        [MaxLength(50)]
        [EmailAddress]
        public string Email { get; set; }
        [MaxLength(250)]
        public string Bio { get; set; }
        [MaxLength(50)]
        public string Company { get; set; }
        public string CompanyId { get; set; }
        [MaxLength(8)]
        public string Title { get; set; }
        [MaxLength(20)]
        public string Fax { get; set; }
        [MaxLength(20)]
        public string Mobile { get; set; }
        [MaxLength(20)]
        public string PanNo { get; set; }
        [MaxLength(20)]
        public string Phone { get; set; }
        [MaxLength(100)]
        [Url]
        public string Website { get; set; }
        public byte Status { get; set; }
        public string UpdatedBy { get; set; }
        public DateTime? UpdatedOn { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedOn { get; set; }
        public ICollection<SupplierAddressModel> Addresses { get; set; }
    }
    [Serializable]
    public class SupplierAddressModel : AddressModel
    {
        public string SupplierId { get; set; }
    }
}
